vulcan.yaml

```diff
@@ -25,6 +25,29 @@ conf:
   # Number of checks to run concurrently
   concurrency: 5
 
+policies:
+  - name: default
+    checks: 
+      - type: vulcan-seekret
+      - type: vulcan-zap
+        options: 
+          max_scan_duration: 3
+      - type: vulcan-trivy
+
+  - name: fast-scan
```

Comment:
Could you use this policy in script/test.sh?

Reply:
Done here [bcde30a](https://github.com/adevinta/vulcan-local/pull/5/commits/bcde30a2eb8af290d35c60a0eaf772b5c92d46c4).